### DISCUSSION TITLE
test262 results

### DISCUSSION THREAD

**Discussion by nicolo-ribaudo** [zloirock/core-js#1260] [General]
Hi :)

We have been running test262 tests with Babel + core-js + Node.js 0.10 for a while, and now the results are published at https://test262.fyi/. The test runner we use is not perfect so some results are false positives/negatives, but it might be interesting for you to find bugs in core-js.

**Comment by zloirock** [comment]
Thanks. I do the same with `test262-harness` (usually, in more recent versions Node versions), but I didn't run this test case for some months. I'll take a look.

**Reply by zloirock** [reply]
It's strange to see, for example, 50% here:

![image](https://github.com/zloirock/core-js/assets/2213682/2ffd1a9d-1738-4c56-b5be-420c601e650a)

IIRC the problems were only in 1 assert of hundreds that's impossible to fix without full implementation of `JSON.parse`. As an option, here not disabled some asserts that depend on some modern non polifillable / non transpilable features that are not directly related to this proposal.

Similar in the rest cases.
